source/DSCResources/DSC_CMClientStatusSettings/DSC_CMClientStatusSettings.schema.mof

```diff
@@ -0,0 +1,12 @@
+[ClassVersion("1.0.0"), FriendlyName("CMClientStatusSettings")]
+class DSC_CMClientStatusSettings : OMI_BaseResource
+{
+    [Key, Description("Specifies the SiteCode for the Configuration Manager site.")] String SiteCode;
```

Comment:
Is this a key? Do the sites have different settings that can be set from a different server?

Reply:
This sets the drive letter which is the site that it would make the change to as part of the Import-module and Set-location.  Technically you can connect to a different site, as long as the SCM console is installed (this is what provides the Config Mgr Powershell commands), and have access to the site.